python-cffi/netplan/parser.py

```diff
@@ -13,12 +13,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+from enum import IntEnum
 from typing import Union, IO
 
 from ._netplan_cffi import ffi, lib
 from ._utils import _checked_lib_call
 
 
+class NETPLAN_PARSER_FLAGS(IntEnum):
+    IGNORE_ERRORS = 1
```

Comment:
suggestion: Just call this class `Flags`

We're already in the `netplan.parser` namespace!